Wei.xcodeproj/project.pbxproj

```diff
@@ -936,6 +936,9 @@
 							com.apple.Keychain = {
 								enabled = 1;
 							};
+							com.apple.SafariKeychain = {
+								enabled = 0;
+							};
```

Comment:
Is this needed for URL Schemes?

Reply:
i dont know it was just generated automatically

Reply:
👍